distribution/lib/Standard/Test/0.0.0-dev/src/Main.enso

```diff
@@ -658,3 +693,67 @@ fail_match_on_unexpected_error error frames_to_skip =
     loc = Meta.get_source_location 1+frames_to_skip
     msg = "An unexpected dataflow error (" + payload.to_text + ") has been matched (at " + loc + ")."
     here.fail msg
+
+
+## PRIVATE
+   Write the JUnit XML header.
+wrap_junit_testsuites : Suite_Config -> Any -> Nothing
+wrap_junit_testsuites config ~function =
```

Comment:
```suggestion
wrap_junit_testsuites config ~action =
```

I suggest to call such entities `action`. `function` brings associations with taking arguments but it is not even a 0-argument function but a suspended block of code to be executed.